spinalcordtoolbox/image.py

```diff
@@ -16,7 +16,8 @@
 import warnings
 import logging
 import math
-from typing import Sequence
+from typing import Sequence, Tuple
+from copy import deepcopy
 
```

Comment:
One quick thing to mention: Right now, running QC reports on some sagittal images runs into https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3055, where the output QC report is ridiculously small.

![image](https://user-images.githubusercontent.com/16181459/195652623-67740a48-064b-4c09-92b8-ae0880b1925a.png)


Given that this was only recently fixed, I'm willing to bet that merging `master` into this PR will fix this issue here, too.

Reply:
Yep! Merging `master` has fixed this issue.